Bottom: 9f1b60a77a7e1959d953b293a75a03a3b004ca44
Top:    626fa6167c67ba142887a5dfa93471065387bc60
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-02 16:54:54 +0800

Refresh of refresh-of-tuntap-introduce-tx

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index eb6a47b..6ab12b1 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -2417,6 +2417,7 @@ static int tun_chr_open(struct inode *inode, struct file * file)
 {
 	struct net *net = current->nsproxy->net_ns;
 	struct tun_file *tfile;
+
 	DBG1(KERN_INFO, "tunX: tun_chr_open\n");
 
 	tfile = (struct tun_file *)sk_alloc(net, AF_UNSPEC, GFP_KERNEL,
